docker-compose.production.yaml

```diff
@@ -38,9 +38,9 @@ services:
     image: pgvector/pgvector:pg16
     restart: always
     ports:
-      - '127.0.0.1:5432:5432'
+      - '5432:5432'
```

Comment:
`127.0.0.1:5432:5432`- This is needed because we don't want to expose this port outside localhost for a production deployment.